store/src/java/com/zimbra/cs/redolog/op/CreateVolume.java

```diff
@@ -114,10 +116,14 @@ public void redo() throws Exception {
             }
         }
         try {
+            Volume.StoreType enumStoreType =
+                (1 == storeType) ? Volume.StoreType.FILE_STORE : Volume.StoreType.EXTERNAL;
```

Comment:
I think it should be Volume.StoreType.INTERNAL instead of Volume.StoreType.FILE_STORE